### PR TITLE
Migrated civicrmtheme and civicrm_contact_ref to Backdrop

### PR DESCRIPTION
**Pull request by herbdool**
Tested manually on a default install and both appear to work well.

### CODE REVIEW

File: modules/civicrmtheme/civicrmtheme.install [hunk 1]
```diff
@@ -53,8 +53,40 @@ function civicrmtheme_install() {
...
-  variable_del('civicrmtheme_theme_admin');
-  variable_del('civicrmtheme_theme_public');
+
+}
+
+/**
+ * Implements hook_update_last_removed().
+ */
+function civicrmtheme_update_last_removed() {
+  return 7000;
```
Comment (quicksketch) [inline modules/civicrmtheme/civicrmtheme.install]: Is this correct? We should only have `hook_update_last_removed()` if the D7 version of this module included a `hook_update_7000()`. If the schema version if not 7000, then the update will refuse to run (AFAIK).

**Comment by quicksketch** [comment]
Looking good! We're missing `hook_config_info()` in `civicrmtheme.module`. Otherwise importing and exporting of configuration will not work.

**Comment by herbdool** [comment]
@quicksketch I've pushed my changes. What's the process again? Created a new PR or just use this one?

**Comment by quicksketch** [comment]
You can "fix" the current one by doing a rebase: https://backdropcms.org/develop/pull-requests#clean-requests, but I can just fix this one up.

I squashed the commits into a single one and merged it here: https://github.com/civicrm/civicrm-backdrop/commit/c3aaac987ea72dedd17046eecb35a4594307f602 Thanks!

**Pull request closed without merging**